user-management/src/main/elm/build-app.sh

```diff
@@ -4,4 +4,4 @@
 # whatever the path from which this script is called
 ELM_DIR="$( cd "$( dirname "$0" )" && pwd )"
 cd $ELM_DIR
-elm make --optimize sources/UserManagement.elm --output=generated/UserManagement.js
```

Comment:
why do you remove the --optimize?

Reply:
so he can use Debug functions, but in the end the optimize flag should be readded